Consul/Consul.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>1.6.1.1</VersionPrefix>
+    <Version>1.6.1.2-rc1</Version>
```

Comment:
We could do this instead
```
<Version>1.6.1.2</Version>
<VersionSuffix>rc1</VersionSuffix>
```
So that prepares things for the continuous delivery pipeline that just needs to update `VersionSuffix` ([example](https://github.com/videolan/libvlcsharp/blob/e4e7ff1adcde647184de566fa5bf7d433eed5e3d/buildsystem/build.cake#L106)).
```

Reply:
https://github.com/G-Research/consuldotnet/pull/49/ might conflict